src/server.js

```diff
@@ -115,7 +121,13 @@ const initializeApi = async (port = Config.API_PORT) => {
 const initializeAdmin = async (port = Config.ADMIN_PORT) => {
   await connectDatabase()
   await migrate()
-  const OpenAPISpecPath = Path.resolve(__dirname, './interface/admin-swagger.yaml')
+  let apiPath
+  if (Config.FEATURE_ENABLE_EXTENDED_PARTY_ID_TYPE) {
+    apiPath = './interface/thirdparty/admin-swagger.yaml'
+  } else {
+    apiPath = './interface/admin-swagger.yaml'
+  }
+  const OpenAPISpecPath = Path.resolve(__dirname, apiPath)
```

Comment:
Looks good, although I'd probably generalize this into it's own function... maybe something like:

```js
const OpenAPISpecPath = pathForConfig(Config)
```

`pathForConfig` can check: (1) is this the admin or non-admin API? and (2) should we enable the extended party id type?

Reply:
done.